File: codeup/100000576/B.cpp
#include <iostream>
using namespace std;

int main() {
  int N, M, iter, num_list[200];
  while (scanf("%d", &N) != EOF) {
    // fill(num_list, num_list + 200, 0);
    for (int i = 0; i < N; i++) scanf("%d", &num_list[i]);
    scanf("%d", &M);

    iter = -1;
    for (int i = 0; i < N; i++) {
      if (M == num_list[i]) {
        iter = i;
        break;
      }
    }
    printf("%d\n", iter);
  }

  return 0;
}

File: codeup/100000576/B.py
#!/usr/bin/python3

import sys

while True:
    line = sys.stdin.readline()
    if not line:
        break
    N = int(line)
    num_list = [int(x) for x in input().split()]
    M = int(input())
    iter = -1
    for i in range(N):
        if M == num_list[i]:
            iter = i
            break
    print(iter)
